core/modules/views/tests/views_layout.test

```diff
@@ -0,0 +1,135 @@
+<?php
+/**
+ * @file
+ * Tests for views and layout collab
```

Comment:
I'm not sure what this sentence means either. Maybe use the language you use in the PR: "Tests for context support to Views blocks associated with Layouts." I suggest capitalizing Views and Layouts since they're specific features not generic terms.